formula/src/main/java/com/instacart/formula/internal/ManagerDelegate.kt

```diff
@@ -1,12 +1,20 @@
 package com.instacart.formula.internal
 
-import com.instacart.formula.Transition
-import kotlin.reflect.KClass
+/**
+ * Used by [FormulaManagerImpl] to delegate and request certain actions when it
+ * cannot handle them internally.
+ */
+internal interface ManagerDelegate {
 
-interface ManagerDelegate {
-    fun onTransition(
-        formulaType: KClass<*>,
-        result: Transition.Result<*>,
-        evaluate: Boolean,
-    )
+    /**
+     * When [FormulaManagerImpl] is not currently running and needs to be evaluated again, it
+     * will request the parent delegate to trigger a new evaluation run.
+     */
+    fun requestEvaluation()
```

Comment:
requestEvaluation seems somewhat of subset of `onPendingTransition(transition: DeferredTransition<*, *, *>)`. Can you help me understand the difference?

Reply:
Good point, I was able to remove `onPendingTransition` from the interface and only keep `requestEvaluation`